unlock-app/src/components/interface/checkout/main/Renew.tsx

```diff
@@ -171,6 +176,21 @@ export function Renew({
     }
   }
 
+  const SwitchNetwork = () => (
```

Comment:
I wonder if we should isolate and create a dedicated component?

Reply:
If it's re-used more, we should.